spring-context/src/main/java/org/springframework/ui/ConcurrentModel.java

```diff
@@ -16,14 +16,14 @@
 
 package org.springframework.ui;
 
-import java.util.Collection;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-
 import org.springframework.core.Conventions;
 import org.springframework.lang.Nullable;
 import org.springframework.util.Assert;
 
+import java.util.Collection;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
```

Comment:
Please refrain from making formatting changes like these.

Reply:
Sure, sorry i'll change it right now